app/styles/components/_c-interface.scss

```diff
@@ -0,0 +1,104 @@
+.c-interface {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
If possible, it might be worth changing some of the things in this file (like display: flex, flex-wrap wrap, and padding) to using au classes

Reply:
Seems a bit complicated to me as the responsive behaviour is quite custom here. I guess it's possible but might cause conflicts with utility classes being the most specific.